Tasks/Gradle/CodeAnalysis/Common/BaseTool.ts

```diff
@@ -78,6 +78,21 @@ export abstract class BaseTool implements IAnalysisTool {
         return tl.getBoolInput(this.uiInputName, false);
     }
 
+    protected findHtmlReport(xmlReport: string): string {
```

Comment:
I've changed the code to search in the old way as well as looking for a file called "checkstyle.html", thereby preserving old functionality as well as adding the new search path.